### PR TITLE
Prepare blog post for publishing

### PR DESCRIPTION
**Pull request by meggie-d**
# Summary

Provide a summary of your blog post here.

# Checklist

- [x] I can successfully knit the blog post Rmd file.

- [x] Figures, graphs, tables render properly in the blog post Rmd file.

- [ ] Abbreviations are expanded/explained.

- [ ] The blog post Rmd file header contains the post title, post description, author information, and categories.

- [ ] The blog post content (Rmd file, figures, tables, etc.) does not contains any PHI.

### CODE REVIEW

File: _posts/2022-05-09-ooh-na-na-where-are-my-sodium-labs/the-na-bug.Rmd [hunk 1]
```diff
@@ -30,7 +30,7 @@ library(xts)
 library(dygraphs)
 library(dplyr)
 library(ggplot2)
-data <- read.csv(file.path(here::here(), "_posts/2022-05-02-the-na-bug", "daily_electrolyte_counts.csv"), stringsAsFactors = FALSE) %>%
+data <- read.csv(file.path(here::here(), "_posts/2022-05-09-ooh-na-na-where-are-my-sodium-labs", "daily_electrolyte_counts.csv"), stringsAsFactors = FALSE) %>%
```
Comment (meggie-d) [inline _posts/2022-05-09-ooh-na-na-where-are-my-sodium-labs/the-na-bug.Rmd:33]: When I used the rename function to change the date, it changed it to the title within the Rmd, I can change it back to "the-na-bug" if you'd prefer that!

File: _posts/2022-05-09-ooh-na-na-where-are-my-sodium-labs/the-na-bug.Rmd [hunk 2]
```diff
@@ -6,15 +6,15 @@ author:
   - name: Chloe Pou-Prom
```
Comment (meggie-d) [inline _posts/2022-05-09-ooh-na-na-where-are-my-sodium-labs/the-na-bug.Rmd:6]: Unrelated to this file, but what do you think of adding a PR checklist for the blog post team for when we're publishing? (Could contain some of the same items as the writer's checklist, like ensuring the Rmd knits properly etc.)

Comment (chloepouprom) [reply _posts/2022-05-09-ooh-na-na-where-are-my-sodium-labs/the-na-bug.Rmd:6]: Yes, I think that makes a lot of sense! I think we should either update the PR template to include this checklist: https://github.com/LKS-CHART/blog/blob/main/.github/pull_request_template.md or update the ClickUp document or both

File: _posts/2022-05-09-ooh-na-na-where-are-my-sodium-labs/the-na-bug.Rmd (hunk 1 above)
Comment (chloepouprom) [reply _posts/2022-05-09-ooh-na-na-where-are-my-sodium-labs/the-na-bug.Rmd:33]: I think using the title makes sense

**Review by chloepouprom — APPROVED** [review]

File: _posts/2022-05-09-ooh-na-na-where-are-my-sodium-labs/the-na-bug.Rmd (hunk 2 above)
Comment (meggie-d) [reply _posts/2022-05-09-ooh-na-na-where-are-my-sodium-labs/the-na-bug.Rmd:6]: Sounds good, I will add it to the PR template/clickup doc!

**Pull request merged**